src/main/java/net/sf/jabref/importer/fetcher/GVKParser.java

```diff
@@ -497,8 +485,7 @@ private Element getChild(String name, Element e) {
     }
 
     private String removeSortCharacters(String input) {
```

Comment:
Can you please check that no one calls this code expecting that the input string is changed, i.e simply as `removeSortCharacters(input)` instead of `output = removeSortCharacters(input)`.